lib/tic_tac_toe.ex

```diff
@@ -0,0 +1,13 @@
+defmodule TicTacToe do
+  alias TicTacToe.Core.Game, as: Game
+  alias TicTacToe.Console.Board, as: Board
+
+  def main(_) do
+    game = Game.new()
+    Board.print(game.board)
```

Comment:
I think it's reasonable to format the board here.

```suggestion
    board_string = Board.format(board)
    IO.puts(board_string)
```

Reply:
Right now, I have `Board.print` both formatting a board and  then printing it with `IO.puts` to minimize our calls to `IO.puts` to one spot (in hopes that we can later inject a different dependency). Do you think it's best to remove  `Board.print` and in practice always format a board into a string  and then print it as a second step?

Reply:
Oh I see. I think I misunderstood something on my first read. Ideally, we would call `IO.puts` in exactly one place.

Maybe introduce `lib/tic_tac_toe/console/output.ex`. Something like:

```elixir
defmodule TicTacToe.Console.Output do
  def print(string) do
    IO.puts(string)
  end
end
```

I think I (incorrectly) assumed that `IO.puts` was already the wrapper.